pkg/support/dump_others.go

```diff
@@ -18,15 +18,38 @@
 package support
 
 import (
+	"bytes"
 	"fmt"
 	"path"
 	"path/filepath"
 	"strings"
+	"sync"
+
+	"k8s.io/klog/v2"
 
 	"antrea.io/antrea/pkg/agent/util/iptables"
+	nftclient "antrea.io/antrea/pkg/agent/util/nftables"
 	"antrea.io/antrea/pkg/util/logdir"
 )
 
+// nftablesIPv4Supported and nftablesIPv6Supported check if the kernel supports nftables.
+// They initialize the client once to verify support, but the returned clients are not used.
+var nftablesIPv4Supported = sync.OnceValue(func() bool {
+	if _, err := nftclient.New(true, false); err != nil {
+		klog.ErrorS(err, "nftables IPv4 not supported on this Node")
```

Comment:
I would switch it to `klog.InfoS` - it may be a pretty harmless error in some cases - even if we don't end up logging the actual `err`